### DISCUSSION TITLE
failed to run the example

### DISCUSSION THREAD

**Discussion by Hejie-Guo** [mangerij/ferret#327] [General]
Hello, I download the package and make it to be a git repostory with "git init", and then make the file.

When i run the example, I can run the BTO_wall_T298K.i file，but the other examples show some different problems.
for example, when i run the BTO_90wall*.i, the problems show as following:
"/home/g00667429/projects/ferret-master/examples/domain_walls/BTO_90wall_T298K.i:67.3: unused parameter 'GlobalParams/vol'"

and run the BTO_monodomain_Tdef.i file, problems shown as following:
"*** ERROR ***
A 'LocalABO3EigenstressAux' is not a registered object.

If you are trying to find this object in a dynamically loaded library, make sure that
the library can be found either in your "Problem/library_path" parameter or in the
MOOSE_LIBRARY_PATH environment variable.
"

**Comment by heinono1** [comment]
Hi,

Newer versions of MOOSE throw an error if there are unused global
parameters. You can just comment out the line with the unused global
parameter.

I don't know off-hand the problem with the *EigenstressAux but "not
registered object" indicates that there have been changes/updates to the
aux kernels in Ferret that are not compatible with that input file.

In any case, if the tests run and other examples do as well, I would not
worry about these two.

Cheers,
Olle

**Comment by mangerij** [comment]
Hi @Hejie-Guo 

Unsure how I never got the notifications for the discussion page.
That object probably was pushed and then removed at some point. There is no guarantee that the example inputs work (only tutorials and tests). Since it is an `AuxKernel`, you can just remove it and it should work as intended.

Please email me at johnma@dtu.dk for further assistance.

Best,
John
